static/src/javascripts/projects/common/modules/experiments/tests/acquisitions-epic-liveblog.js

```diff
@@ -27,7 +34,10 @@ const getLiveblogEntryTimeData = el => {
     };
 };
 
-const setEpicLiveblogEntryTimeData = (el, timeData) => {
+const setEpicLiveblogEntryTimeData = (
+    el: Element,
+    timeData: TimeData
+): void => {
     if (!el) {
```

Comment:
I think it is, it was a relic of an earlier approach before a refactor where it was possible for this to be null. I think now because it references an epic we've just inserted it should always be there